storage/google/cloud/storage/bucket.py

```diff
@@ -459,6 +459,9 @@ class Bucket(_PropertyMixin):
     )
     """Allowed values for :attr:`location_type`."""
 
+    RETRIES = 4
```

Comment:
How is this number determined?

Reply:
This number just added to as get it confirmed during the review.
Though I have seen this used in existing code while i was investigating the issue.

4 retries would result is actual wait of 1.x, 2.x, 4.x and 8.x seconds wait on multiple retries respectively. 
x is random milliseconds added during each iteration.